packaging/fedora-25/snap-mgmt.sh

```diff
@@ -23,8 +23,8 @@ if [ "$1" = "purge" ]; then
         umount -l ${SNAP_MOUNT_DIR} || true
     fi
 
-    mounts=$(systemctl list-unit-files --full | grep '^snap[-.].*\.mount' | cut -f1 -d ' ')
-    services=$(systemctl list-unit-files --full | grep '^snap[-.].*\.service' | cut -f1 -d ' ')
+    mounts=$(systemctl list-unit-files --full | grep '^var-lib-snapd-snap[-.].*\.mount' | cut -f1 -d ' ')
+    services=$(systemctl list-unit-files --full | grep '^var-lib-snapd-snap[-.].*\.service' | cut -f1 -d ' ')
```

Comment:
It is. Without that the snap-mgmt.sh script doesn't work. I would had let you know tomorrow about this as this this prevents our 2.26.x based packages from cleaning up properly. All systemd unit files are prefixed with the snap mount dir which is /var/lib/snapd/snap on Fedora.